components/compliance-service/ingest/pipeline/publisher/node_manager_publisher.go

```diff
@@ -90,6 +90,10 @@ func gatherInfoForNode(in message.Compliance) (*manager.NodeMetadata, error) {
 		tags = append(tags, &common.Kv{Key: "environment", Value: in.Report.GetEnvironment()})
 	}
 
+	mgrType := in.Report.GetAutomateManagerType()
+	if in.Report.GetSourceFqdn() != "" {
```

Comment:
if there's a value for the chef server, we know the node is managed by chef

Reply:
If the value is "localhost" it is not managed by a chef server it is an effortless or chef solo report.

Reply:
whaddya think? should we distinguish between chef-server managed and just chef solo/effortless?

Reply:
I don't think we should for now. But it looks like could when and if we want to.